Firestore/core/src/firebase/firestore/core/view_snapshot.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_VIEW_SNAPSHOT_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_VIEW_SNAPSHOT_H_
+
+namespace firebase {
+namespace firestore {
+namespace core {
+
```

Comment:
The objc had this comment:
```
/**	
 * The types of changes that can happen to a document with respect to a view.	
 * NOTE: We sort document changes by their type, so the ordering of this enum is significant.	
 */
```

If that's still relevant, it should be ported. (Especially the second sentence.)